docs/api/api-specification.md

```diff
@@ -0,0 +1,2170 @@
+---
+layout: default
+title: Interface Specification
+parent: API
+nav_order: 4
+---
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+## API Specification
+
+For the purposes of this documentation, when describing an API route, [base] includes `/fhir/r4`.
+JSON is currently the only supported format. Please make use of the `application/fhir+json` mime type for the Accept header. The Content-Type header must also correspond to this mime type.
+
+<a name="data-representation"/>
+
+### Data Representation
+Because the Sara Alert API follows the FHIR specification, there is a mapping between known kinds of Sara Alert data and their associated FHIR resources.
+
+| Sara Alert                | FHIR Resource |
+| :---------------          | :------------ |
+| Monitoree                 | [Patient](https://hl7.org/fhir/R4/patient.html)|
+| Monitoree Lab Result      | [Observation](https://hl7.org/fhir/R4/observation.html)|
+| Monitoree Daily Report    | [QuestionnaireResponse](https://www.hl7.org/fhir/questionnaireresponse.html)|
+
+<a name="supported-scopes"/>
+
+### Supported Scopes
+For applications following the [SMART-on-FHIR App Launch Framework "Standalone Launch" Workflow](#standalone-launch), these are the available scopes:
+
+* `user/Patient.read`,
+* `user/Patient.write`,
+* `user/Patient.*`, (for both read and write access to this resource)
+* `user/Observation.read`,
+* `user/QuestionnaireResponse.read`,
+
+For applications following the [SMART on FHIR Backend Services Workflow](#backend-services), these are the available scopes:
+
+* `system/Patient.read`,
+* `system/Patient.write`,
+* `system/Patient.*`, (for both read and write access to this resource)
+* `system/Observation.read`,
+* `system/QuestionnaireResponse.read`,
+
+Please note a given application and request for access token can have have multiple scopes, which must be space-separated. For example:
+```
+`user/Patient.read system/Patient.read system/Observation.read`
+```
+
+<a name="cap"/>
+
+### CapabilityStatement and Well-Known Uniform Resource Identifiers
+
+<a name="cap-get"/>
+
+A capability statement is available at `[base]/metadata`:
+
+#### GET `[base]/metadata`
+
+<details>
+  <summary>Click to expand JSON snippet</summary>
+
+```json
```

Comment:
This doesn't appear to be formatting as we would want. The generated results look like this:
![image](https://user-images.githubusercontent.com/52747882/102381551-5e525e00-3f97-11eb-80af-214009d85f44.png)

Reply:
Yeah I guess jekyll requires a ` <div markdown="1">` to wrap any markdown that you're nesting inside of HTML. I made this change in a bunch of places on the walkthrough page but failed to do it for the spec page. Good catch

Reply:
Fixed here and everywhere else on this page https://github.com/SaraAlert/SaraAlert/pull/598/commits/490bef24961bf80cb728c31cd3185f0d063a4d0b